xnmt/eval/tasks.py

```diff
@@ -110,6 +110,8 @@ class AccuracyEvalTask(EvalTask, reports.Reportable, Serializable):
     model: generator model to generate hypothesis with
     eval_metrics: list of evaluation metrics (list of Evaluator objects or string of comma-separated shortcuts)
     inference: inference object
+    perform_inference: Whether to generate the output or not. One eval task can use difference hyp_file that is generated
```

Comment:
``difference hyp_file that is generated`` -> ``an already existing hyp_file that was generated`` ?

Reply:
Done!